dpctl/tests/test_usm_ndarray_ctor.py

```diff
@@ -59,13 +59,13 @@ def test_allocate_usm_ndarray(shape, usm_type):
 
 
 def test_usm_ndarray_flags():
-    assert dpt.usm_ndarray((5,)).flags == 3
-    assert dpt.usm_ndarray((5, 2)).flags == 1
-    assert dpt.usm_ndarray((5, 2), order="F").flags == 2
-    assert dpt.usm_ndarray((5, 1, 2), order="F").flags == 2
-    assert dpt.usm_ndarray((5, 1, 2), strides=(2, 0, 1)).flags == 1
-    assert dpt.usm_ndarray((5, 1, 2), strides=(1, 0, 5)).flags == 2
-    assert dpt.usm_ndarray((5, 1, 1), strides=(1, 0, 1)).flags == 3
+    assert dpt.usm_ndarray((5,)).flags.flags == 3
```

Comment:
Same this here, `_flags.__equal__` can implement the logic for comparing to integers, as well as to another `_flags` object.

But most like the test must change, e.g.

```python
assert dpt.usm_ndarray((5, 2)).flags["C"]
assert dpt.usm_ndarray((5, 2), order="F").flags["F"]
assert dpt.usm_ndarray((5, 1, 1), strides=(1, 0, 1)).flags["FC"]
```